tanzu-build-service/install-tbs.hbs.md

```diff
@@ -335,40 +281,6 @@ To install `full` Tanzu Build Service dependencies:
 
     Where `VERSION` is the version of the Tanzu Build Service package you retrieved earlier.
 
-## <a id="auto-updates-config"></a> (Optional) Configure automatic dependency updates
```

Comment:
Removal of `<a id="auto-updates-config" />` will break the link on [line 29](https://github.com/pivotal/docs-tap/blob/1ac74b9e0c554542a6a7eafd51026c67f4810f96/tanzu-build-service/install-tbs.hbs.md?plain=1#L29) of `install-tbs.hbs.md` (this file) and the link on [line 9](https://github.com/pivotal/docs-tap/blob/1ac74b9e0c554542a6a7eafd51026c67f4810f96/multicluster/reference/tap-values-build-sample.hbs.md?plain=1#L94) of `multicluster/reference/tap-values-build-sample.hbs.md`.

Reply:
oh good catch i need to remove it from that file too